tests/baselines/reference/doubleUnderscoreSymbolsAndEmit.errors.txt

```diff
@@ -0,0 +1,60 @@
+tests/cases/compiler/index.tsx(33,1): error TS2308: Module "./b" has already exported a member named '__foo'. Consider explicitly re-exporting to resolve the ambiguity.

```

Comment:
Too late - already split this file into about 4 unique tests which test difference things with double underscore prefixes. 🐱